Add documentation for BIP21 features (#23)
breez/breez-sdk-liquid-docs: src/guide/send_payment.md

# Sending Lightning/Liquid Payments

Once the SDK is set up, you can start sending payments too.

> The `destination` field of the payment request now supports Liquid BIP21/plain addresses and invoices. For onchain (Bitcoin) payments, see [here](pay_onchain.md).

Reviewer: ```suggestion
The `destination` field of the payment request supports Liquid BIP21, Liquid addresses and Lightning invoices. For onchain (Bitcoin) payments, see [Sending an on-chain transaction](pay_onchain.md).
```

> **Note:** There are cases in which both the destination and the user provide an amount to be paid. Here are the possible scenarios and their outcomes:
> - In case both the Liquid BIP21 URI and the request amount field are set, the SDK will make sure the two values match, else an error will be thrown.
> - In case both the invoice amount and the request amount field are set, the SDK will make sure the two values match, else an error will be thrown.
> - In case the user attempts to pay a plain Liquid address, the request amount field **must** be present. Else, the "Amount Missing" error will be thrown.

Reviewer: Good, but I think needs to be a bit cleaner
```suggestion
## Amount validation
There are cases in which both the destination and the user provide an amount to be paid. Here are the possible scenarios and their outcomes:
- **Liquid BIP21 URI amount and request amount field are set** - the SDK will make sure the two values match, else an error will be thrown.

- **Invoice amount and request amount field are set** - the SDK will make sure the two values match, else an error will be thrown.

- **Liquid address is used** - the SDK will make sure the request amount is set. Else, the "Amount Missing" error will be thrown.

```
